src/components/card/card.component.jsx

```diff
@@ -0,0 +1,43 @@
+import "./card.style.css";
+/**
+ *
+ * @param {string} title - The title of the Card
+ * @param {string} size - The Size of card expected value either small or big 
+ * @param {object} data - The data will be displayed in card it will be represented as key: value 
+ * if data has one prop it will display its value only
+ 
+ */
```

Comment:
This comment is a bit misleading. The component doesn't need 3 parameters it only takes one which is `props` but we need to tell other programmers what is contained inside these props so they can pass the correct types so we write it like this
```js
/**
 * @param {Object} props
 * @param {String} props.title
 * @param {Object<string, string>} props.data
 * @param {"small" | "big"} props.size
 */
 ```
 so when hovering over the component this appears 
 
![image](https://github.com/ASU-Semicolon/The-New-Dashboard/assets/80486237/2402859d-24b4-465f-abfa-ab2ebf487fb6)